README.md

```diff
@@ -1,11 +1,13 @@
 puppet-lint-anchor-check
 ========================
 
-[![Build Status](https://img.shields.io/travis/voxpupuli/puppet-lint-anchor-check.svg)](https://travis-ci.org/voxpupuli/puppet-lint-anchor-check)
-[![Gem Version](https://img.shields.io/gem/v/puppet-lint-anchor-check.svg)](https://rubygems.org/gems/puppet-lint-anchor-check)
-[![Gem Downloads](https://img.shields.io/gem/dt/puppet-lint-anchor-check.svg)](https://rubygems.org/gems/puppet-lint-anchor-check)
-[![Coverage Status](https://coveralls.io/repos/github/voxpupuli/puppet-lint-anchor-check/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppet-lint-anchor-check?branch=master)
-[![Dependency Status](https://gemnasium.com/badges/github.com/voxpupuli/puppet-lint-anchor-check.svg)](https://gemnasium.com/github.com/voxpupuli/puppet-lint-anchor-check)
+[![License](https://img.shields.io/github/license/voxpupuli/puppet-lint-spaceship_operator_without_tag-check.svg)](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/blob/master/LICENSE)
+[![Test](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/test.yml/badge.svg)](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/test.yml)
+[![Release](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/release.ym
+l)
+[![RubyGem Version](https://img.shields.io/gem/v/puppet-lint-spaceship_operator_without_tag-check.svg)](https://rubygems.org/gems/puppet-lint-spaceship_operator_without_tag-check)
+[![RubyGem Downloads](https://img.shields.io/gem/dt/puppet-lint-spaceship_operator_without_tag-check.svg)](https://rubygems.org/gems/puppet-lint-spaceship_operator_without_tag-check)
+[![Donated by Alexander Fisher](https://img.shields.io/badge/donated%20by-Alexander%20Fisher-fb7047.svg)](#copyright)
```

Comment:
puppet-lint-anchor-check is unexpectedly changed to puppet-lint-spaceship_operator_without_tag-check